cluster/images/etcd/rollback/README.md

```diff
@@ -0,0 +1,41 @@
+# Rollback workflow
+
+Build it in this directory.
+Make sure you have etcd dependency ready. Last time we use etcd v3.0.7.
```

Comment:
@xiang90 - thanks
I would like to update to 3.0.7 when we incorporate rollback to the script there. I will take care of it.
@lavalamp @timothysc - FYI